scripts/bz.py

```diff
@@ -6,6 +6,12 @@
 import platform
 import json
 import glob
+import argparse
+
+parser = argparse.ArgumentParser()
```

Comment:
`argparse` looks nice when we use several CLI flags seems, Thanks for using it :tada: